tests/unit/stores/utilities/pool.spec.js

```diff
@@ -296,4 +272,377 @@ describe('pool', () => {
+              },
+            ],
+          },
+        },
+      },
+      {
+        id: '11877',
+        type: 'tubes',
+        attributes: {
+          barcode: 'TRAC-2-11877',
+        },
+        relationships: {
+          libraries: {
+            data: {
+              type: 'libraries',
+              id: '14160',
+            },
+          },
+        },
+      },
+      {
+        id: '11876',
+        type: 'tubes',
+        attributes: {
+          barcode: 'TRAC-2-11876',
+        },
+        relationships: {
+          libraries: {
+            data: {
+              type: 'libraries',
+              id: '14159',
+            },
+          },
+        },
+      },
+    ]
+
+    const requests = [
+      {
+        id: '8951',
+        type: 'requests',
+        attributes: {
+          library_type: 'Pacbio_HiFi',
+          sample_name: 'DTOL15016450',
+          barcode: 'FS71986093',
+          source_identifier: 'FS71986093',
+        },
+        relationships: {},
+      },
+      {
+        id: '8950',
+        type: 'requests',
+        links: {
+          self: 'https://traction.psd.sanger.ac.uk/v1/pacbio/requests/8950',
+        },
+        attributes: {
+          library_type: 'Pacbio_HiFi',
+          sample_name: 'DTOL15016449',
+          barcode: 'FS71986813',
+          source_identifier: 'FS71986813',
+        },
+        relationships: {},
+      },
+    ]
+
+    const aliquots = [
+      {
+        id: '35493',
+        type: 'aliquots',
+        attributes: {
+          aliquot_type: 'derived',
+          source_id: 14159,
+          source_type: 'Pacbio::Library',
+          used_by_id: 6011,
+          used_by_type: 'Pacbio::Pool',
+          state: 'created',
+          volume: 7.5,
+          concentration: 12.9,
+          insert_size: 9933,
+          template_prep_kit_box_barcode: '036037102141700123124',
+          tag_id: 600,
+        },
+        relationships: {
+          tag: {
+            data: {
+              type: 'tags',
+              id: '600',
+            },
+          },
+          pool: {
+            data: {
+              type: 'pools',
+              id: '6011',
+            },
+          },
+        },
+      },
+      {
+        id: '35494',
+        type: 'aliquots',
+        attributes: {
+          aliquot_type: 'derived',
+          source_id: 14160,
+          source_type: 'Pacbio::Library',
+          used_by_id: 6011,
+          used_by_type: 'Pacbio::Pool',
+          state: 'created',
+          volume: 7.5,
+          concentration: 12.2,
+          insert_size: 9360,
+          template_prep_kit_box_barcode: '036037102141700123124',
+          tag_id: 601,
+        },
+        relationships: {
+          tag: {
+            data: {
+              type: 'tags',
+              id: '601',
+            },
+          },
+          pool: {
+            data: {
+              type: 'pools',
+              id: '6011',
+            },
+          },
+        },
+      },
+    ]
```

Comment:
Do we need to keep this data over here or can it be moved to factory?

Reply:
@stevieing can we get the data from the factories instead of using fixture here ? we are not testing the factories here, so I think it should be fine, or may be I am missing something ?

Reply:
I did think about that but the factories use these methods so it would not make sense. It was also really difficult to make sense of what was going on using the factory data.

I think that these kind of methods sit better in the aliquot code so we can look at it as part of our review. I have noticed it all seems to relate to aliquot technical debt which it would be good to reduce and maybe remove these methods altogether.

I have cut down the data as much as possible and it was much easier to test like this as these methods are quite confusing.